packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx

```diff
@@ -97,7 +97,6 @@ export const LoginForm: React.FunctionComponent<LoginFormProps> = ({
           label={rememberMeLabel}
           checked={isRememberMeChecked}
           onChange={onChangeRememberMe}
-          aria-label={rememberMeAriaLabel || rememberMeLabel}
```

Comment:
This looks good to me - let's add one more comment up above in rememberMeAriaLabel (to the comment section noting this propType is now deprecated and is no longer used in the code.  Also, please open an issue with a breaking change label indicating we should delete this propType in a future breaking change release.

Reply:
Just pushed again with this addition to the comment. Should be all set now -- made a new breaking change issue as well.